### PR TITLE
story/BODS-578: Easier to understand operator profiles detailed export column names

### PR DESCRIPTION
**Pull request by ndemenezes**

### CODE REVIEW

File: transit_odp/organisation/tests/csv/test_service_codes.py [hunk 1]
```diff
@@ -329,251 +331,293 @@ def test_csv_output():
...
+
+    assert csv_output["row3"][0] == '"PD0000099:3"'
+    assert csv_output["row3"][1] == '"line1 line2"'
+    assert csv_output["row3"][2] == '"Yes"'
+    assert csv_output["row3"][3] == '"Published"'
+    assert csv_output["row3"][4] == '"Registered"'
+    assert csv_output["row3"][5] == '"In Scope"'
+    assert csv_output["row3"][6] == '"In Season"'
+    assert csv_output["row3"][7] == '"OTC variation not published"'
+    assert csv_output["row3"][8] == '"1"'
```
Comment (alvak1) [inline transit_odp/organisation/tests/csv/test_service_codes.py:450]: Getting an error when running the tests

**Review by alvak1 — COMMENTED** [review]
comment added

File: transit_odp/organisation/tests/csv/test_service_codes.py (hunk 1 above)
Comment (ndemenezes) [reply transit_odp/organisation/tests/csv/test_service_codes.py:450]: @alvak1 What is the error you are getting? (just ran the test cases again and they seem to be passing)

Comment (alvak1) [reply transit_odp/organisation/tests/csv/test_service_codes.py:450]: image

**Review by alvak1 — COMMENTED** [review]
added comment

File: transit_odp/organisation/tests/csv/test_service_codes.py (hunk 1 above)
Comment (ndemenezes) [reply transit_odp/organisation/tests/csv/test_service_codes.py]: Resolved this @alvak1

**Review by pritish-ranjan01 — APPROVED** [review]

**Pull request merged**